.travis/remove-all-bot-comment.js

```diff
@@ -0,0 +1,29 @@
+let github = require('octonode');
+
+var client = github.client(process.env.GITHUB_TOKEN);
+
+async function getBotInfo() {
+  let ghme = client.me();
+  const result = await ghme.infoAsync();
+  return result[0];
+}
+
+var ghpr = client.issue(process.env.TRAVIS_REPO_SLUG, process.env.TRAVIS_PULL_REQUEST);
```

Comment:
please use let.

and I don'lt understand the name again.
what gh stands for?
what is ghpr?
Pr is for pull request?